src/runtime/node/url/index.ts

```diff
@@ -84,33 +107,46 @@ export const format = <typeof url.format>(
 
 // https://nodejs.org/api/url.html#url_url_domaintoascii_domain
 // TODO
-export const domainToASCII = <typeof url.domainToASCII>function (domain) {
+export const domainToASCII = <typeof nodeUrl.domainToASCII>function (domain) {
   return domain;
 };
 
 // https://nodejs.org/api/url.html#url_url_domaintounicode_domain
 // TODO
-export const domainToUnicode = <typeof url.domainToUnicode>function (domain) {
-  return domain;
-};
+export const domainToUnicode = <typeof nodeUrl.domainToUnicode>(
+  function (domain) {
+    return domain;
+  }
+);
 
 // https://nodejs.org/api/url.html#url_url_pathtofileurl_path
 // TODO
-export const pathToFileURL = <typeof url.pathToFileURL>function (path) {
+export const pathToFileURL = <typeof nodeUrl.pathToFileURL>function (path) {
   return new URL(path);
 };
 
 // https://nodejs.org/api/url.html#url_url_fileurltopath_url
 // TODO
-export const fileURLToPath = <typeof url.fileURLToPath>function (url) {
+export const fileURLToPath = <typeof nodeUrl.fileURLToPath>function (url) {
   if (typeof url === "string") {
     url = new URL(url);
   }
   return url.pathname;
 };
 
-export default <typeof url>{
+export const resolveObject = function resolveObject(
+  source: string,
+  relative: string,
+) {
+  if (!source) return relative;
+  // Node.js logic: return urlParse(source, false, true).resolveObject(relative);
+  return new URL(relative, source).toString();
+};
+
+export default {
   URL,
+  // @ts-expect-error
```

Comment:
can you add a reason for the exception so that we know when it's safe to remove it? is this a typings issue?

Reply:
Yeah and i guess it would be never typed since is deperecated...